sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -120,30 +113,13 @@ static const struct dmi_system_id sof_rt5682_quirk_table[] = {
 		.driver_data = (void *)(SOF_RT5682_MCLK_EN |
 					SOF_RT5682_SSP_CODEC(1)),
 	},
-	{
-		/*
-		 * Dooly is hatch family but using rt1015 amp so it
-		 * requires a quirk before "Google_Hatch".
-		 */
-		.callback = sof_rt5682_quirk_cb,
-		.matches = {
-			DMI_MATCH(DMI_SYS_VENDOR, "HP"),
-			DMI_MATCH(DMI_PRODUCT_NAME, "Dooly"),
-		},
-		.driver_data = (void *)(SOF_RT5682_MCLK_EN |
-					SOF_RT5682_SSP_CODEC(0) |
-					SOF_SPEAKER_AMP_PRESENT |
-					SOF_RT1015_SPEAKER_AMP_PRESENT |
-					SOF_RT5682_SSP_AMP(1)),
```

Comment:
CML chromebooks use "Google_Hatch" quirk which amplifier is MAX98357A so we added a quirk for Dooly to use RT1015 amp. Since we could detect the amp type now, this quirk could be removed.

Reply:
> CML chromebooks use "Google_Hatch" quirk which amplifier is MAX98357A so we added a quirk for Dooly to use RT1015 amp. Since we could detect the amp type now, this quirk could be removed.

Thanks for the explanation. So, Dooly will use "Google_Hatch" quirk.